tests/JsonTest.php

```diff
@@ -238,13 +238,26 @@ public function convertsGuzzleResponseExceptionsIntoFriendlyClientExceptions()
      */
     public function throwsInvalidJsonResponse()
     {
-        $this->mockClient()->request('GET', $this->url('/foo'), ['query' => []])
+        $this->mockClient()->request('GET', $this->url('/foo'), ["query" => []])
             ->willReturn($this->mockResponseBody('invalid')->reveal());
 
         $this->client->get('/foo');
     }
 
     /**
+     * @test
+     */
+    public function passesOnHeaders()
+    {
+        $this->mockClient()->request('GET', $this->url('/foo'), ['headers' => ['Content-Type' => 'application/json; charset=utf-8'], "query" => []])
```

Comment:
The above line is a bit too long and hard to read, IMO.
What do you think of...?

```php
$options = [
    'headers' => ['Content-Type' => 'application/json; charset=utf-8'],
    'query' => []
];
$this->mockClient()->request('GET', $this->url('/foo'), $options)
```